src/slices/Image/ImagesInColumns.tsx

```
@@ -4,38 +4,70 @@ import {
-      {isFilled.group(images) &&
-        images.map(({ image }, i) => (
+export const ImagesInColumns: React.FC<ImagesInColumnsProps> = ({ images }) => {
+  if (!isFilled.group(images)) return;
+
+  const sectionCount = Array(Math.ceil(images.length / 2)).fill(null);
+
+  return sectionCount.map((_, index) => {
+    const [imageIndex1, imageIndex2] = [2 * index, 2 * index + 1];
+
+    return (
+      <div className="max-w-6xl mx-auto grid md:grid-cols-2 gap-12 items-center mb-12 last:mb-0">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)